### PR TITLE
Added orange-earl-grey-iced-tea

### PR DESCRIPTION
**Pull request by sujoy-coder**
Nice and cool project!
Hopefully you like my contribution and add it, If so please add `hacktoberfest-accepted` tag. 

Thank You :)

### CODE REVIEW

File: src/recipes/orange-earl-grey-iced-tea.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            "quantity": "1/4",
+            "measure": "cup",
+            "ingredient": "suger"
+        }
+    ],
+    "directions": [
+        "Steep loose tea (or tea bags) and orange peel in boiling water for 3 to 5 minutes",
+        "Strain the tea (or remove tea bags and orange peel) and pour into a large pitcher. Stir in orange juice and sugar until the sugar is dissolved. Add cold water. Refrigerate until chilled, about 2 hours. Serve over ice with orange wedges, if desired"
+    ],
+    "image": "<drink-name.jpg>",
```
Comment (alfg) [inline src/recipes/orange-earl-grey-iced-tea.json:41]:
```suggestion
    "image": "orange-earl-grey-iced-tea.jpg",
```

**Review by alfg — CHANGES_REQUESTED** [review]
Hi @sujoy-coder, I added a suggested fix for the image path in the JSON file. Also, please provide an image that isn't stretched as 600x400px in size. Thanks.

**Comment by alfg** [comment]
Hi @sujoy-coder, the image still appears to be stretched. Can you provide an image while maintaining the aspect ratio?

**Comment by sujoy-coder** [comment]
@alfg sir, I don't understand what is happening although I make the image 600x400 px dimension as mentioned. So I make it 600x600 px . Now check it pls...

Thank You...

**Comment by alfg** [comment]
This is fine. Thank you!

**Comment by sujoy-coder** [comment]
Thank You....sir :)

**Pull request merged**